project/pom.xml

```diff
@@ -1120,19 +1120,6 @@
             <links>
               <link>http://download.oracle.com/javase/6/docs/api/</link>
             </links>
-            <footer><![CDATA[
-            <!-- Google Analytics -->
```

Comment:
Out of curiosity: do we know whether there was ever a GA account associated with this?

Reply:
If we have it we never used it. I think Everett added this?

Reply:
Looks like Adrian in 0e07b7236fb594146bb8c3cd485004716f2897c0.